LION/models/learned_regularizer/ACR.py

```diff
@@ -61,7 +97,7 @@ class L2net(nn.Module):
     def __init__(self):
         super(L2net, self).__init__()
 
-        self.l2_penalty = nn.Parameter((-9.0) * torch.ones(1))
+        self.l2_penalty = nn.Parameter((-36.0) * torch.ones(1))
```

Comment:
This is true always, or just for 2DeteCT? i.e. should I make it a parameter?

Reply:
Why 512?

Reply:
Oh that was just kind fine-tuned for 512x512 images